arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.series

```diff
@@ -0,0 +1,27 @@
+# Kconfig - ST Microelectronics STM32L0 MCU line
+#
+# Copyright (c) 2017 RnDity Sp. z o.o.
+# Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_SERIES_STM32L0X
+
+source "arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.stm32l0*"
+
+config SOC_SERIES
+	default stm32l0
+
+if GPIO_STM32
+
+config GPIO_STM32_PORTD
```

Comment:
I'm not sure about this ports. L0s have ports A-E and H, and some of them have even less.